src/overlays/actors/ovl_Bg_Mori_Hashigo/z_bg_mori_hashigo.h

```diff
@@ -6,9 +6,16 @@
 
 struct BgMoriHashigo;
 
+typedef void (*BgMoriHashigoActionFunc)(struct BgMoriHashigo*, GlobalContext*);
+
 typedef struct BgMoriHashigo {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x84];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ ColliderJntSph collider;
+    /* 0x0184 */ ColliderJntSphItem colliderItems[1];
+    /* 0x01C4 */ BgMoriHashigoActionFunc actionFunc;
+    /* 0x01C8 */ s16 hitTimer;
+    /* 0x01CA */ s16 bounceCounter;
```

Comment:
Not sure about this:

```suggestion
    /* 0x0164 */ ColliderJntSph claspCollider;
    /* 0x0184 */ ColliderJntSphItem claspColliderItems[1];
    /* 0x01C4 */ BgMoriHashigoActionFunc actionFunc;
    /* 0x01C8 */ s16 claspHitTimer;
    /* 0x01CA */ s16 ladderBounceCounter;
```
I do think it makes it clearer (especially the collider being clasp-only) but it may harm readability?

`InitCollider` could also be renamed to make clasp-only explicit, as well as `sJntSphInit` and `sJntSphItemsInit`